Uno.Gallery/Uno.Gallery.Shared/Views/Shell.xaml

```diff
@@ -179,7 +179,11 @@
 			   TargetType="MenuFlyoutItem">
```

Comment:
@Xiaoy312 Just a few notes while I test the azure static web link for this PR:
- When clicking on an item from one of the menus, it will open a separated tab. But if you come back to the galley app tab and try again to open one of the menu, you will notice that there is no background anymore:
![image](https://github.com/unoplatform/Uno.Gallery/assets/16295702/40ce5485-baec-431e-a792-b47d5ef197bd)

Does not seem related to your current changes as I am able to reproduce it with the current prod version of Gallery. 
But as you are currently updating the wasm banner, not sure if it is something quick to fix?
If it is not, we will open an issue (if there is none already)

Reply:
not that i am aware of, but i havent looked into really.
it is opened already: unoplatform/uno#17366